debian/index.js

```diff
@@ -140,3 +136,29 @@ ipcMain.on('player2Win', (e, args) => {
     win.webContents.send('player2Win', args)
   } catch(err) { /* window already closed */ }
 })
+
+// Code based on MarshallOfSound Google-Play-Music-Desktop-Player-UNOFFICIAL-
+// file src/main/features/linux/mediaKeysDBus.js
+function registerBindings(desktopEnv, bus) {
+  bus.getInterface(`org.${desktopEnv}.SettingsDaemon`,
+  `/org/${desktopEnv}/SettingsDaemon/MediaKeys`,
+  `org.${desktopEnv}.SettingsDaemon.MediaKeys`, (err, iface) => {
+    if (!err) {
+      iface.on('MediaPlayerKeyPressed', (n, keyName) => {
+        switch (keyName) {
+          case 'Next': win.webContents.executeJavaScript(`
+                         window.electronConnector.emit('play-next')
+                       `); return;
+          case 'Previous': win.webContents.executeJavaScript(`
+                             window.electronConnector.emit('play-previous')
+                           `); return;
+          case 'Play': win.webContents.executeJavaScript(`
+                         window.electronConnector.emit('play-pause')
+                       `); return;
+          default: return;
+        }
+      });
+      iface.GrabMediaPlayerKeys(0, `org.${desktopEnv}.SettingsDaemon.MediaKeys`);
+    }
+  });
+};
```

Comment:
I don't really know what arg1,arg2,arg3 mean. maybe you can give them a better name